include/cassandra.h

```diff
@@ -131,6 +131,11 @@ typedef struct CassString_ {
 #define CASS_INET_V6_LENGTH 16
 
 /**
+ * The size of an inet string including a null terminator.
+ */
+#define CASS_INET_STRING_LENGTH 46
```

Comment:
No change requested, but just an aside:
I kind of liked the way they defined this size with no 'magic numbers' in the old branch of libuv:
https://github.com/libuv/libuv/blob/v0.10/src/inet.c#L81

It's removed in 1.0, though.